internal/compiler-bridge/src/main/scala/xsbt/DelegatingReporter.scala

```diff
@@ -27,29 +36,33 @@ private object DelegatingReporter {
     override def toString =
       (sourcePath0, line0) match {
         case (Some(s), Some(l)) => s + ":" + l
-        case (Some(s), _)       => s + ":"
-        case _                  => ""
+        case (Some(s), _) => s + ":"
+        case _ => ""
```

Comment:
why are these not aligned?

NB when changing scalafmt config you can't just reformat again - you need to format from the original source code, then the result will correctly reflect the specified configuration.